### PR TITLE
WPW-143: check if parent is alive and exit if not

### PR DESCRIPTION
**Pull request by wberendt**
- rpc agent will check every 3 s. whether the parent process is alive, and will exit if not

### CODE REVIEW

File: applications/rpc-agent/main.go [hunk 1]
```diff
@@ -177,6 +184,23 @@ func startRPC() {
...
+		if alive {
+			time.Sleep(time.Second * 3)
+			continue
+		}
+		if err != nil {
+			log.Debug("IsProcAlive error:", err)
+		}
+		break
+	}
+	log.Debug("Parent process is no longer alive, exiting...")
```
Comment (maciej-wakula-mobica) [inline applications/rpc-agent/main.go]: Isn't DEBUG too low?
IMHO this should be at least INFO (or even FATAL if we consider this an extraordinary scenario which normally does not happen and restarting rpc-agent would not help in resolving the issue)

File: applications/rpc-agent/main.go [hunk 2]
```diff
@@ -177,6 +184,23 @@ func startRPC() {
 	log.Debug("End startRPC()")
 }
 
+func scanParent() {
```
Comment (maciej-wakula-mobica) [inline applications/rpc-agent/main.go]: Can we have some test for that?

Comment (wberendt) [reply applications/rpc-agent/main.go]: It's privet function, so it's not required.

File: applications/rpc-agent/main.go (hunk 1 above)
Comment (wberendt) [reply applications/rpc-agent/main.go]: Debug is ok, it's info, for developers.

**Pull request merged**